share/spice/quandl/fundamentals/quandl_fundamentals.css

```diff
@@ -0,0 +1,74 @@
+/*
+    Document   : 
+    Created on : 2015-02-12 08:21 PM
+    Author     : brianrisk
+    Description: styles based on the 'Stocks' spice for visual consistency
+*/
+
+#quandl h1.quandl-header {
```

Comment:
I'd prefer a simple class name to the id or the crazy regex selector.  (`.quandl` instead of `#quandl`). I really don't foresee any need for us to create a `.quandl` class internally so there's no legitimate risk of conflict. Overall I don't think it'd hurt anything and it does make sense to remove the tab-specific selectors to make the styling more reusable.